### DISCUSSION TITLE
Understanding Bioinformatic Outputs/Analyses

### DISCUSSION THREAD

**Discussion by skreling** [sr320/course-fish546-2021#23] [General]
I think I just need to do a lot more reading, but is there any particular resource you would recommend for understanding the different types of analyses (I know there are myriad options for this, but I don't really know where to begin). I feel like I don't even know what I don't know at this point. I know for my personal research I'll be doing metabarcoding, and conceptually I understand how this works, but in understanding the specific data cleaning and analysis steps I'll have to take to get to a finished product is a bit overwhelming for me and not really sure where to start.

**Comment by kubu4** [comment]
Just documenting discussion from class today.

>  a bit overwhelming

Yep! It definitely is!!

The first part is to evaluate what your data is (e.g. sequence files) and decide how to run some quality control on them to see how they look. This will be addressed in the next week or two.

Please feel free to continue posting issues in the class repo. Even questions like: "I have this file. What do I do with it?"
